game/board.py

```diff
@@ -0,0 +1,22 @@
+from game.cell import Cell
+from constans.constants_game import (
+    LARGE
+)
+
+
+class Board():
+    def __init__(self) -> None:
+        self._board = [
+            [Cell(i, j) for j in range(LARGE)] for i in range(LARGE)
```

Comment:
nit: It has more sense if you give names to variables. (Please check if the names orders are okay)
```suggestion
            [Cell(row, col) for col in range(LARGE)] for row in range(LARGE)
```